spec/taxon_breadcrumbs_spec.rb

```diff
@@ -125,8 +125,8 @@
 
         expect(breadcrumbs).to eq(
           breadcrumbs: [
-            { title: "Home", url: "/" },
-            { title: "Parent A", url: "/parent-a" },
+            { title: "Home", url: "/", is_page_parent: false },
+            { title: "Parent A", url: "/parent-a", is_page_parent: true },
             { title: "Taxon", is_current_page: true },
```

Comment:
As discussed, govuk_navigation_helpers always returns a breadcrumb with `is_parent_page: true`. Static can handle breadcrumbs without it (because it was missing from older versions from this library), but this helper now always adds it so there's no extra case to test here.